### DISCUSSION TITLE
A search bug when search a keyword in a repository.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37784] [Repositories]
I'm trying to search: socket-timeout in repository yt-dlp/yt-dlp. It says could't find anything about that.<p>
<img width="1008" alt="Screenshot_29" src="https://user-images.githubusercontent.com/37928893/198879636-4a90000a-5f25-425b-b89d-62cce0e2d143.png">
But there should be a result in file : yt_dlp/options.py
<img width="573" alt="Screenshot_27" src="https://user-images.githubusercontent.com/37928893/198879718-78aa69ff-d731-44d3-929e-c73e11ae337e.png">
And the above picture is also the result that searched by keyword: socket.timeout
So I think it's a bug of searching function.

**Comment by ghost** [comment]
And I also tried "socket\\-timeout". It returned the same result.

**Comment by airtower-luna** [comment]
Can confirm the problem. Interestingly searching `"--socket-timeout"` works, so I suspect the bug is related to word boundaries.

**Reply by ghost** [reply]
> Can confirm the problem. Interestingly searching `"--socket-timeout"` works, so I suspect the bug is related to word boundaries.

It doesn't work on my PC. My browser is chrome 106.0.5249.119
